src/woo_publications/publications/api/serializers.py

```diff
@@ -119,11 +129,10 @@ class DocumentSerializer(serializers.ModelSerializer):
         max_length=1,  # pyright: ignore[reportCallIssue]
     )
     eigenaar = EigenaarSerializer(
-        source="get_owner",
         label=_("owner"),
-        help_text=_("The creator of the document, derived from the audit logs."),
+        help_text=_("The creator of the document."),
         allow_null=True,
-        read_only=True,
+        required=False,
```

Comment:
I realize we need to make this writable so that you can change the owner, but it's probably a good idea to amend the help text to say that if provided, this overrides the values taken from the audit headers and if empty-ish, the values will be populated from the audit headers.